config.go

```diff
@@ -3,31 +3,38 @@ package fangs
 import (
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/anchore/go-logger"
 	"github.com/anchore/go-logger/adapter/discard"
 )
 
 type Config struct {
-	Logger  logger.Logger `yaml:"-" json:"-" mapstructure:"-"`
-	AppName string        `yaml:"-" json:"-" mapstructure:"-"`
-	TagName string        `yaml:"-" json:"-" mapstructure:"-"`
-	File    string        `yaml:"-" json:"-" mapstructure:"-"`
-	Finders []Finder      `yaml:"-" json:"-" mapstructure:"-"`
+	Logger                 logger.Logger `yaml:"-" json:"-" mapstructure:"-"`
+	AppName                string        `yaml:"-" json:"-" mapstructure:"-"`
+	TagName                string        `yaml:"-" json:"-" mapstructure:"-"`
+	ConfigureMultipleFiles bool          `yaml:"-" json:"-" mapstructure:"-"`
+	InheritMultipleFiles   bool          `yaml:"-" json:"-" mapstructure:"-"`
+	File                   string        `yaml:"-" json:"-" mapstructure:"-"`
+	Files                  []string      `yaml:"-" json:"-" mapstructure:"-"`
+	Finders                []Finder      `yaml:"-" json:"-" mapstructure:"-"`
+	ProfileKey             string        `yaml:"-" json:"-" mapstructure:"-"`
+	Profiles               []string      `yaml:"-" json:"-" mapstructure:"-"`
 }
 
 var _ FlagAdder = (*Config)(nil)
 
 // NewConfig creates a new Config object with defaults
 func NewConfig(appName string) Config {
 	return Config{
-		Logger:  discard.New(),
-		AppName: appName,
-		TagName: "mapstructure",
+		Logger:                 discard.New(),
+		AppName:                appName,
+		TagName:                "mapstructure",
+		ConfigureMultipleFiles: true,
+		InheritMultipleFiles:   true,
```

Comment:
By defaulting this to `false`, I think this change would be mostly compatible from the end-user perspective, even if we got rid of the single `File` property, because the same `--config` or `-c` would work as expected